utils/eval_report_parsing.py

```diff
@@ -80,8 +81,118 @@ def coallate_results(release_directory_path, config):
+                                            continue
+                                        if category not in experiment_json["categories"]:
+                                            experiment_json["categories"].append(category)
+                                        if category not in model_scores:
+                                            model_scores[category] = []
+                                        if type(scores[category]) == float:
+                                            score = scores[category]
+                                        else:
+                                            if name == "Geometric Reasoning (GeoMeter)": # geometric reasoning reports count instead of percentage
+                                                none = 0
+                                                if "none" in scores and (scores["none"] == scores["none"]): # check for NaN
+                                                    none = scores["none"]
+                                                score = scores[category]["correct"] * 1.0 / (scores[category]["correct"] + scores[category]["incorrect"] + none)
+                                            else:
+                                                score = scores[category]["correct"]
+                                        model_scores[category].append(score)
+                                break
+                            except FileNotFoundError:
+                                print("Error finding " + file_path)
+                                continue
+                            
+                        if model == 'GPT-4o_2024_05_13_450K':
+                            model = 'GPT-4o-2024-05-13'
+                        if model == 'GPT-4o_2024_05_13':
+                            model = 'GPT-4o-2024-05-13'
+                        if model == "LLaVA-34B":
+                            model = "Llava-1_6-34B"
+                        if model == "GPT-4":
+                            model = "GPT-4-1106-Preview"
+
+                        if len(model_scores) == 0:
+                            print("No scores found for model: " + model + " in experiment: " + experiment_json["title"])
+                            continue
+                        scores = []
+                        for category in experiment_json["categories"]:
+                            scores.append(round(math.fsum(model_scores[category])  * 100.0 / len(model_scores[category]), 1))
+                        graph_json["values"].append({
+                            "name": model,
+                            "scores": scores
+                        })
+                experiment_json["series"].append(graph_json)
+            data[name]["experiments"].append(experiment_json)
+    with open('website/static/benchmark_results.json', 'w') as f:
+        json.dump(data, f, indent=2)
+
+
 # Example usage
 release_directory_path = '\\release'

```

Comment:
should we update the path here too ? to '/'?

Reply:
going to leave as is for now, we can change if/when there's more minor edits required.